src/kasse.c

```diff
@@ -147,51 +147,16 @@ static signed int buy(char *name, unsigned int price) {
   BYTE c, x, y, nickname_len;
   int einheiten;
   char nickname[NICKNAME_MAX_LEN + 1];
-  char rest[11];
+  char rest[10];
   struct credits_t *credit;
 
   memset(nickname, '\0', sizeof(nickname));
-  memset(rest, ' ', sizeof(rest));
```

Comment:
Removing the padding will destroy the alignment on the physical print out: https://github.com/nnev/c128-kasse/blob/b6fc2f7cbcd8bd7cb5aa9e83a358a91cc0396158/src/kasse.c#L134-L137

Reply:
All strings that have a known maximum length could/should be printf()d using %Ns. This should save us from manual padding.

Reply:
This will be completely reworked with the coming binlog.